src/shared/utils/exportData/parseOptions.test.ts

```diff
@@ -0,0 +1,213 @@
+            optionId: '8747ed38-f4b3-4fde-855e-20471ca0aa8e',
+            score: 2,
+            alert: null,
+            value: 1,
+          },
+          {
+            optionId: 'd79a44c2-78b6-4fdd-b6fa-8bf41d0b3760',
+            score: 4,
+            alert: null,
+            value: 2,
+          },
+        ],
+      },
+    ],
+  };
+  const multiSelectRows = {
+    rows: [
+      {
+        id: '1248a537-218e-4285-99ca-d435a79fea39',
+        rowName: 'row1',
+        rowImage: null,
+        tooltip: null,
+      },
+    ],
+    options: [
+      {
+        id: '95fdad7a-a94b-46ca-907c-5ef858df8535',
+        text: 'opt1',
+        image: null,
+        tooltip: null,
+      },
+      {
+        id: '6c1aa535-34f3-424f-bde4-40d937b0013b',
+        text: 'opt1',
+        image: null,
+        tooltip: null,
+      },
+    ],
+    dataMatrix: [
+      {
+        rowId: '1248a537-218e-4285-99ca-d435a79fea39',
+        options: [
+          {
+            optionId: '95fdad7a-a94b-46ca-907c-5ef858df8535',
+            score: 3,
+            alert: null,
+            value: 1,
+          },
+          {
+            optionId: '6c1aa535-34f3-424f-bde4-40d937b0013b',
+            score: 5,
+            alert: null,
+            value: 2,
+          },
+        ],
+      },
+    ],
+  };
+  const sliderRows = {
+    rows: [
+      {
+        minLabel: 'min',
+        maxLabel: 'max',
+        minValue: 1,
+        maxValue: 5,
+        minImage: null,
+        maxImage: null,
+        scores: [1, 2, 3, 4, 5],
+        alerts: null,
+        id: '87ff7de0-3552-4a14-b748-d6bcb12f4d87',
+        label: 'slider',
+      },
+    ],
+  };
+  const drawing = {
+    drawingExample:
+      'https://media-dev.cmiml.net/mindlogger/391962851007982489/91c9d624-3ab1-45ca-8f68-4cfc94cdd195/Transfer Ownership - 1.png',
+    drawingBackground:
+      'https://media-dev.cmiml.net/mindlogger/391962851007982489/3374a78e-0a46-4add-b89e-d5bff79e9678/Transfer Ownership - 1.png',
+  };
+  const audio = {
+    maxDuration: 300,
+  };
+  const audioPlayer = {
+    file: 'https://media-dev.cmiml.net/mindlogger/391962851007982489/b6d8573c-8174-41cb-b0fa-ad9fb0d237ff/t-rex-roar.mp3',
+  };
+
+  test.each`
+    responseType                                | responseValues      | expected                                                                                                  | description
+    ${ItemResponseType.SingleSelection}         | ${single}           | ${'Opt1: 1 (score: 4), Opt2: 2 (score: 2)'}                                                               | ${'single'}
+    ${ItemResponseType.MultipleSelection}       | ${multi}            | ${'Opt1: 1 (score: 2), Opt2: 2 (score: 1)'}                                                               | ${'multi'}
+    ${ItemResponseType.Slider}                  | ${slider}           | ${'0: 0 (score: 1), 1: 1 (score: 2), 2: 2 (score: 3), 3: 3 (score: 4), 4: 4 (score: 5), 5: 5 (score: 6)'} | ${'slider'}
```

Comment:
could you pls move some big expected values to variables tom make the test.each table smaller?